systemtests/src/test/java/io/enmasse/systemtest/PlansTest.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2018, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.enmasse.systemtest;
+
+import io.enmasse.systemtest.amqp.AmqpClient;
+import io.enmasse.systemtest.bases.TestBase;
+import io.enmasse.systemtest.resources.AddressPlan;
+import io.enmasse.systemtest.resources.AddressResource;
+import io.enmasse.systemtest.resources.AddressSpacePlan;
+import io.enmasse.systemtest.resources.AddressSpaceResource;
+import io.enmasse.systemtest.standard.QueueTest;
+import io.enmasse.systemtest.standard.TopicTest;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.slf4j.Logger;
+
+import java.util.*;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertEquals;
+
+@Category(IsolatedAddressSpace.class)
+public class PlansTest extends TestBase {
```

Comment:
Why this file is not in some package?

Reply:
this class is in io.enmasse.systemtest; package (it doesn't match any of currently created packages and it doesn't make sense for me to create new package only for this class. Maybe once we'll have more Plans related tests then we could create new package for them) do you agree?

Reply:
Maybe call it PlansTestBase

Reply:
In general, I think we should try to not rely too much on inheritance for this kind of things. It would be better if we moved to junit5 IMO to get better tools to create these kind of 'test fixtures'.

Reply:
but it's not 'Base' because each class with new plans should add @setup and @teardown because java doesn't support multi-inheritance and we would like to use it in WebConsole tests for example. Basically is it the same approach like is already used with SeleniumProvider in WebConsole tests.